components/PlayButton/PlayButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, ButtonHTMLAttributes } from "react";
+import Icon from "../Icon/Icon";
+import "./PlayButton.module.scss";
+
+type Props = ButtonHTMLAttributes<HTMLButtonElement> & {
+  onClick: () => void;
+};
+
+const PlayButton: React.FC<Props> = ({ onClick, ...buttonProps }) => {
```

Comment:
It's common to use `...rest` for the rest props :) https://mortenbarklund.com/blog/react-typescript-props-spread/
```suggestion
const PlayButton: React.FC<Props> = ({ onClick, ...rest }) => {
```